src/Collector/Collector/ResolvedNodeCollector.php

```diff
@@ -42,16 +42,16 @@ public function getNodeType(): string
      */
     public function processNode(Node $node, Scope $scope): ?array
     {
-        if ($this->lattePhpDocResolver->resolveForNode($node, $scope)->isIgnored()) {
-            return null;
-        }
         $resolvedNodes = [];
         foreach ($this->latteTemplateResolvers as $latteTemplateResolver) {
             $resolvedNode = $latteTemplateResolver->collect($node, $scope);
             if ($resolvedNode !== null) {
                 $resolvedNodes[] = $resolvedNode;
             }
         }
+        if (count($resolvedNodes) > 0 && $this->lattePhpDocResolver->resolveForNode($node, $scope)->isIgnored()) {
```

Comment:
This was moved because I found out that calling it for every node (even when no resolver matches it) can have  memory and performace impact